resolwe_bio/processes/workflows/bbduk_star_featurecounts.yml

```diff
@@ -0,0 +1,377 @@
+            the second read on the opposite strand. In strand-specific reverse
+            assay these rules are reversed.
+        - name: feature_class
+          label: Feature class
+          type: basic:string
+          default: exon
+          hidden: '!quantification.show_advanced'
+          description: >
+            Feature class (3rd column in GTF/GFF3 file) to be used. All other
+            features will be ignored.
+        - name: feature_type
+          label: Feature type
+          type: basic:string
+          default: gene
+          hidden: '!quantification.show_advanced'
+          choices:
+            - label: gene
+              value: gene
+            - label: transcript
+              value: transcript
+          description: >
+            The type of feature the quantification program summarizes over
+            (e.g. gene or transcript-level analysis). The value of this
+            parameter needs to be chosen in line with 'ID attribute' below.
+        - name: id_attribute
+          label: ID attribute
+          type: basic:string
+          default: gene_id
+          allow_custom_choice: true
+          hidden: '!quantification.show_advanced'
+          choices:
+            - label: gene_id
+              value: gene_id
+            - label: transcript_id
+              value: transcript_id
+            - label: ID
+              value: ID
+            - label: geneid
+              value: geneid
+          description: >
+            GTF/GFF3 attribute to be used as feature ID. Several GTF/GFF3 lines
+            with the same feature ID are considered as parts of the same
+            feature. The feature ID is used to identify the counts in the
+            output table. In GTF files this is usually 'gene_id', in GFF3 files
+            this is often 'ID', and 'transcript_id' is frequently a valid
+            choice for both annotation formats.
+  run:
+    language: workflow
+    program:
+      - id: preprocessing
+        run: bbduk-single
+        input:
+          reads: '{{ input.preprocessing.reads }}'
+          reference:
+            sequences: '{{ input.preprocessing.adapters }}'
+          processing:
+            kmer_length: '{{ input.preprocessing.kmer_length }}'
+            hamming_distance: '{{ input.preprocessing.hamming_distance }}'
+          operations:
+            k_trim: r
+            min_k: '{{ input.preprocessing.min_k }}'
+            quality_trim: r
+            trim_quality: '{{ input.preprocessing.trim_quality }}'
+            min_length: '{{ input.preprocessing.min_length }}'
+            trim_pairs_evenly: t
+            trim_by_overlap: t
+      - id: alignment
+        run: alignment-star
+        input:
+          reads: '{{ steps.preprocessing }}'
+          genome: '{{ input.alignment.star_index }}'
+          filtering:
+            outFilterType: BySJout
+            outFilterMultimapNmax: 20
```

Comment:
In this specific workflow, it might make sense to set this parameter to 1, however, Its [not wrong](http://seqanswers.com/forums/showthread.php?t=41991) to leave it at 20. The upside if a more versatile .bam if other tools will potentially be used in downstream analysis. The downside is a bit larger .bam that holds information not useful for FeatureCounts.

Reply:
I followed your suggestion and changed the default value of this parameter to 1. Initially, all parameters were copied from `workflow-bbduk-star-htseq`.